### PR TITLE
Contribution.md file updated

### LINKED ISSUE #250: Update contributing.md
Make the [How can you make your pull request](https://github.com/Real-Dev-Squad/website-crypto/blob/develop/CONTRIBUTING.md#how-can-you-make-your-first-pull-request) section similar to this ->
https://github.com/Real-Dev-Squad/website-www/blob/develop/CONTRIBUTE.md
> don't include point no.7

### PR DESCRIPTION
**Pull request by DeRaowl**
updated contribution.md file with required changes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -43,7 +45,11 @@
...
 ```
 
-# How can you make your first **Pull Request**
+# How can you make a **Pull Request** (PR)
+
+If you get an issue assigned to you, consider following steps mentioned below.
+
+**Note**: Steps 1, 2 and 3 are **one-time** steps required for setup. If you have already cloned the repo and added upstream, consider following this documentation from step 4.
 
 1. **Forking repository**
```
Comment (rajat-mehra05) [inline CONTRIBUTING.md:54]: Can you add `description` for `Forking repository` for this website from the reference given along with the image ?
Like this but for `crypto-site`:

image

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -100,30 +126,35 @@ git add .
 git commit -m "Write message about your commit"
```
Comment (rajat-mehra05) [inline CONTRIBUTING.md:126]: Don't we require to add this command before the commit message `git add .` ?

**Review by rajat-mehra05 — CHANGES_REQUESTED** [review]
Please review the comments, do the needful and resolve the conversation afterwards.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -100,30 +126,35 @@ git add .
...
-For example, if the branch name is develop:
+Once we're in our branch, we **rebase** our branch on top of the current develop branch (we change the base of our branch, so that it appears as if we have worked from the time the latest changes were merged in the develop branch). To do so:
 
 ```
-git merge develop
+git rebase develop
+```
+
+You should solve the merge conflicts, if any.
 ```
```
Comment (akshay1502) [inline CONTRIBUTING.md]: Do we need ``` here?
You can check the preview of markdown file by following [this](https://code.visualstudio.com/docs/languages/markdown#_markdown-preview).
Firefox_Screenshot_2022-02-03T12-01-18 781Z

**Review by rajat-mehra05 — APPROVED** [review]
Looks good to me.

**Review by akshay1502 — APPROVED** [review]

**Comment by akshay1502** [comment]
Thanks for the PR @DeRaowl .

**Pull request merged**